.ci/install_kubernetes.sh

```diff
@@ -18,15 +18,38 @@ cidir=$(dirname "$0")
 source /etc/os-release || source /usr/lib/os-release
 kubernetes_version=$(get_version "externals.kubernetes.version")
 
-if [ "$ID" != "ubuntu" ]; then
-        echo "Currently this script only works for Ubuntu. Skipped Kubernetes Setup"
+if [ "$ID" != "ubuntu" ] && [ "$ID" != "centos" ]; then
+        echo "Currently this script does not work on $ID. Skipped Kubernetes Setup"
         exit
 fi
 
+if [ "$ID" == "ubuntu" ]; then
 sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
```

Comment:
I think there is some indent whitespace breakage now there is an if/else wrapper - pls check.
Also a bunch of extra blank lines I think we can drop.